UDPClientGUI/MyUDPClient.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPCommGUI
+{
+    class MyUDPClient : IClientInterface
```

Comment:
MyUDPClient is better named as "SiceUDPClient" - not a major thing.

Reply:
Changed as commented.